etc/Linux.mk

```diff
@@ -59,70 +63,108 @@ CAT              = /bin/cat
 MV               = /bin/mv
 RM               = /bin/rm
 MKDIR            = /bin/mkdir
-FIND             = /usr/bin/find
-SED              = /bin/sed
-ifeq ($(findstring Ubuntu,$(shell uname -v)),Ubuntu)
-AWK              = /usr/bin/awk
-else			 
-AWK              = /bin/awk
-endif			 
-TR               = /usr/bin/tr
+FIND             = /usr/bin/env find
```

Comment:
I think just `FIND             = find` and letting path resolution work should be just as good as using `/usr/bin/env` here? I can't see why it would need an absolute path

Reply:
yeah, thats probably unnecessary on linux. I just feel a bit better when its obvious which version of a tool is being run and removing any weird user environment vagaries..